k8s/basic/variables.tf

```diff
@@ -1,13 +1,3 @@
-variable "create_app_namespace" {
-  type    = bool
-  default = true
-}
-
-variable "app_namespace" {
-  type    = string
-  default = "default"
-}
```

Comment:
@dzhlobo I'm not sure about an intention of these variables but as I understand they're unused. Let me know if we need them for something.